client/src/components/infoDrawer/infoFormat.js

```diff
@@ -80,20 +82,47 @@ const renderOrganism = (organism, skeleton) => {
 };
 
 // Render list of metadata attributes found in categorical field
-// Ignores categories with empty or null values
 const renderSingleValueCategories = (singleValueCategories, skeleton) => {
   if (singleValueCategories.size === 0) return null;
   return (
     <>
       <H3 className={skeleton ? Classes.SKELETON : null}>Dataset Metadata</H3>
       <UL>
-        {Array.from(singleValueCategories).map((pair) => {
-          if (!pair[1] || pair[1] === "") return null;
+        {Array.from(singleValueCategories).map((pair, index, array) => {
+          const [category, value] = pair;
+          let ontologyTermID = "";
+          // If the value is empty skip it
+          if (!value || value === "") return null;
+
+          // If the next category is a ontology term, let's save it's value to render now
```

Comment:
what's the rationale behind checking the next item in the current iteration instead of just wait for the next iteration to check it?

Reply:
If we tried to handle it on the next iteration we wouldn't be able to iterate of the list using Map since we have to append the value of that checked item to the element we're currently constructing.  AFAIK we don't have access to the resulting array in `Array.prototype.map()`

Reply:
ooh should we use `.reduce()`?

Reply:
Oh great idea!